### DISCUSSION TITLE
The next video in my folder / playlist never plays from the beginning

### DISCUSSION THREAD

**Discussion by Yamilcfg** [AndreyPavlenko/Fermata#167] [Q&A]
Hello guys!

I have Fermata Auto 1.8.11 running in my Galaxy Note 10+ with Android 12 and One UI 4.1 and use the app with Android Auto 7.6.621714. I have a local folder in my phone with music videos and also created a playlist based on this folder. Here is my main problem with Fermata Auto, when I play a video in the folder and I wanto play the next video, when I move to the next video using the button on Android Auto or in my phone, the next video never plays from the beggining. In other words, If I'm playing videos from a folder or playlist and I move to the next video (file), the next video never plays from the beggining, the video starts from a point but never from the beggining. Its like Fermata remember the last position and apply the same position to every video in the folder, and this happen even If I don't move to the next video, when the video finishes and automatically open the next video in the playlist (folder) the next video plays from a ramdom point, but not from the beggining. 

What I would like to do is just to play videos normally, If I don't want to keep listening to a video and wants to move to the next video, I would like the next video to play from the beggining (as normally expected) and not from a point in the middle of the video or even the end. 

What can I do? I really like Fermata Auto but this issue is really annoying, how can I solve this?? .... Thanks in advance

**Comment by AndreyPavlenko** [comment]
If less than 95% of video is watched, Fermata remembers the video position and starts playing from this position next time. You can change this behavior in Settings > Video settings > Video threshold. To disable, set the value to 0%. You can also change this setting for a particular folder - long press on a folder > Playback settings > Video threshold.

**Reply by Yamilcfg** [reply]
Thanks for your reply .......... I already did that, I have the Video Threshold set to zero, in both the general settings and per folder setting and still the same, nothing changes .... its really weird, if I'm watching a video lets say I'm at the 50% of the video and move to the next video, the next video will start at 50%. The same happens when the video reach the ends and moves automatically to the next video in the folder, the video never starts from the beginning .......... I tried uninstalling the app, restarting the phone, installed the app again, nothing ..... tried clearing the app cache, nothing ..... tried every possible option in settings, nothing ..... I really like the app, but this bug is reallt bad

**Comment by AndreyPavlenko** [comment]
Try long press on a video > Mark as watched

**Reply by Yamilcfg** [reply]
And how this is going to help me? ..... If I have a local video folder or playlist and I want to play all the videos in the folder, every time I skip to the next video, or when the current video ends, the next video in the folder / playlist won't start from the beginning .... do I have to mark every video as watched? we are talking about a folder with more than 100 videos

**Comment by AndreyPavlenko** [comment]
You need to do it only once, to reset the saved position. If you have set threshold to 0, the position will not be saved again. As a workaround - rename the folder.

**Reply by Yamilcfg** [reply]
OK, Thanks .......... I'll try that and let you know

**Reply by Yamilcfg** [reply]
That didn't work ..... I marked some videos as watched and renamed the folder, nothing, the same keeps happening

**Reply by AndreyPavlenko** [reply]
Have you set the video threshold to 0 after the folder renaming? Long press on the folder > Playback settings.

**Reply by Yamilcfg** [reply]
Let me give you more details .... for instance, If I start watching a video and I skip to the next video on 0:01 position, the next video will start playing at 0:01 position ...... iF I'm at 02:33 position and I skip to the next video, the next video will start at 02:33 .... If the video reach the end, the next video seems to start in the last known position ...... I installed the app through AAAD

**Reply by Yamilcfg** [reply]
Yes, the video threshold is set to zero, in both the folder settings and general settings

**Reply by AndreyPavlenko** [reply]
Hm... I've never seen such behavior. How do you skip to next? Do you press the next button in the control panel?

**Reply by Yamilcfg** [reply]
Yes, the same behavior happens when I press the next button on the menu bar in the app, or the next button on my steering wheel on my car

**Reply by AndreyPavlenko** [reply]
Does it happen on phone, when not connected to car?

**Reply by Yamilcfg** [reply]
Not sure if you saw my reply yesterday ....... this is happening when not connected to my car, and when connected, both ..... have you found any solution or workaround?

**Comment by Yamilcfg** [comment]
Yes, in both the phone and the car
